components/channel_header/index.js

```diff
@@ -70,7 +70,7 @@ function makeMapStateToProps() {
         } else if (channel && channel.type === General.GM_CHANNEL) {
             gmMembers = doGetProfilesInChannel(state, channel.id, false);
         }
-        const stats = getCurrentChannelStats(state) || {member_count: 0, guest_count: 0, pinnedpost_count: 0};
+        const stats = getCurrentChannelStats(state) || {member_count: 0, guest_count: 0, pinnedpost_count: 0, files_count: 0};
```

Comment:
Since this is a `mapStateToProps` that continually gets recalled, could we define the empty zero stats outside of the function so that we always pass in a stable identity for the zero case? Something like

```js
const emptyChannelStats = {member_count: 0, guest_count: 0, pinnedpost_count: 0, files_count: 0};
...
function makeMapStateToProps() {
   ... 
   return function mapStateToProps() { 
       ...
       const stats = getCurrentChannelStats(state) || emptyChannelStats
       ...
   }
}
```